jujugui/static/gui/src/app/app.js

```diff
@@ -1240,13 +1240,14 @@ YUI.add('juju-gui', function(Y) {
     */
     _generateSocketUrl: function(callback) {
       this.jem = null;
-      if (this.get('jemUrl')) {
+      if (this.get('jemURL')) {
```

Comment:
This is now inconsistent with `jemUrl` in juju.js. It's a little confusing how they refer to the same value but are different. Could you please change this one back or that one forward?